Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/ProjectStatsController.java

```diff
@@ -110,8 +153,10 @@ public String showProjectStatistics(
                 int cnt = getCount(conceptsWithCount);
                 jArray = getTopConceptsJson(conceptsWithCount.subList(0, cnt),
                         cnt);
-                String jsonData = jArray.toString();
-                model.addAttribute("jsonstring", jsonData);
+
+                String jsonData = jArray.toString(); 
+                model.addAttribute("labelCount", jsonData);
+
                 model.addAttribute("networkid", "\"\"");
 
             } catch (JSONException e) {

```

Comment:
the exceptions are all over the place. some you catch like here, some you just let your controller throw. Just catch all of them and show an error message as already done below.